ci-conda.Dockerfile

```diff
@@ -231,6 +231,9 @@ RUN <<EOF
 pip install codecov-cli==${CODECOV_VER}
 EOF
 
+# Install packaging lib
+RUN pip install packaging
```

Comment:
Since this is for the `ci-conda` image, I think it'd be better to use `conda` to install this. That tends to lead to less surprises than mixing `pip` and `conda` installed things in the same environment.

Instead of a new layer, could you just add this to this block?

https://github.com/rapidsai/ci-imgs/blob/0b910783e9ba8dd7e17b123a972f46875883c2e0/ci-conda.Dockerfile#L196-L207

I just double-checked, and it's literally called `packaging` on `conda-forge` too (https://github.com/conda-forge/packaging-feedstock).